### PR TITLE
Issue465 thermal bridge

### PR DESCRIPTION
**Pull request by rubenbaetens**
For review: Inclusion of linear thermal bridge model in window model.

### CODE REVIEW

**Comment by rubenbaetens** [comment]
I should also update the definition of `Q_design`.

File: IDEAS/Buildings/Components/Examples/package.order [hunk 1]
```diff
@@ -2,4 +2,3 @@ BuildingShadeExample
 ZoneExample
 WindowDynamics
 WindowLinearisation
-CavityWalls
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Examples/package.order]: Is this intentional?

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo [hunk 2]
```diff
@@ -18,8 +18,8 @@ partial model PartialSurface "Partial model for building envelope component"
   parameter Modelica.SIunits.Temperature TRef_a=291.15
     "Reference temperature of zone on side of propsBus_a, for calculation of design heat loss"
     annotation (Dialog(group="Design heat loss"));
-  parameter Boolean linearise_a=false
-    "= true, if convective heat transfer should be linearised."
+  parameter Boolean linearise_a=true
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: should stay false

File: IDEAS/Buildings/Components/Examples/package.order (hunk 1 above)
Comment (rubenbaetens) [inline IDEAS/Buildings/Components/Examples/package.order]: I just noticed it too while opening Dymola, will figure out what hapened here.

File: IDEAS/Buildings/Components/Window.mo [hunk 3]
```diff
@@ -45,6 +53,13 @@ model Window "Multipane window"
         extent={{10,-10},{-10,10}},
         rotation=-90,
         origin={-40,-100})));
+
+  Modelica.Thermal.HeatTransfer.Components.ThermalConductor theBri(final G=briType.G) if
+       briType.present "Themal bridge of the window perimeter"
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Window.mo]: Would it not be better to recompute the value of the existing thermal resistor based on the value of `briType.G`? The current implementation is computationally more difficult and adds a graphic element that may better be avoided.

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo (hunk 2 above)
Comment (rubenbaetens) [inline IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: This was unintentional (but weird, cause i can't remember changing this)

**Comment by Mathadon** [comment]
Can you also add a unit test / example that uses thermal bridges?

File: IDEAS/Buildings/Components/Window.mo (hunk 3 above)
Comment (rubenbaetens) [inline IDEAS/Buildings/Components/Window.mo]: That would work for a wall (which also includs thermall mass of a bridge), but very complex for a window.

File: IDEAS/Fluid/BaseCircuits/PumpSupply.mo [hunk 4]
```diff
@@ -23,9 +23,10 @@ model PumpSupply
       addPowerToMedium=false,
       allowFlowReversal=true),                  final useBalancingValve=true,
     balancingValve(show_T=true),
-    booleanInput = true);
+    booleanInput = true,
```
Comment (GlennReynders) [inline IDEAS/Fluid/BaseCircuits/PumpSupply.mo]: Does this belong in this pull request?

File: IDEAS/Buildings/Components/Window.mo [hunk 5]
```diff
@@ -250,6 +267,11 @@ equation
     connect(heaCapGla.port, layFra.port_a) annotation (Line(points={{16,-12},{16,
             -12},{16,80},{10,80}},  color={191,0,0}));
   end if;
+  connect(theBri.port_b, layFra.port_a) annotation (Line(points={{10,40},{16,40},
```
Comment (GlennReynders) [inline IDEAS/Buildings/Components/Window.mo]: Is there a reason why the ports have been switched? I suggest to keep same convention within 1 component.

Comment (GlennReynders) [inline IDEAS/Buildings/Components/Window.mo]: @Mathadon For unittest, do I make new example or can we add additional window type to IDEAS/Buildings/Components/Examples/WindowDynamics?

File: IDEAS/Fluid/BaseCircuits/PumpSupply.mo (hunk 4 above)
Comment (rubenbaetens) [inline IDEAS/Fluid/BaseCircuits/PumpSupply.mo]: This comes from another branch `RBA_bugfixes` which has been merged in

File: IDEAS/Buildings/Components/Window.mo (hunk 5 above)
Comment (rubenbaetens) [inline IDEAS/Buildings/Components/Window.mo]: What has been switched ?

**Comment by rubenbaetens** [comment]
Addressed in #474.

**Pull request closed without merging**